src/Tribe/Views/V2/Hooks.php

```diff
@@ -687,4 +691,7 @@ public function live_filters_maybe_convert( $value ) {
 
 		return $return_value;
 	}
+
+	protected function control_query( \WP_Query $query ) {
```

Comment:
:no_entry_sign: Missing doc comment for function control_query()

<sub>hash: 8614e44de948f476b8930d6d3e29ec0b</sub>